resource-service/src/main/java/uk/gov/gchq/palisade/service/resource/model/AuditableResourceResponse.java

```diff
@@ -90,7 +90,7 @@ public AuditErrorMessage getAuditErrorMessage() {
          * @return the composed immutable object
          */
         public static IResourceResponse create() {
-            return AuditableResourceResponse::new;
+            return (resourceResponse1, auditErrorMessage1) -> new AuditableResourceResponse(resourceResponse1, auditErrorMessage1);
```

Comment:
can this be reverted?

Reply:
Yes, done, but I think it will complain in SonarQube